src/osyris/plot/wrappers.py

```diff
@@ -28,7 +28,7 @@ def quiver(ax,
            cblabel=None,
            density=1,
            color="w",
-           zorder=2,
+           zorder=3,
```

Comment:
I think the bug is that the LIC should have `zorder=1`, instead of changing the `zorder` in `quiver` and `streamplot`